camera_1:
  extrinsic_matrix:
  - - -0.4424574350277336
    - 0.8961382816553698
    - 0.03417013813901948
    - 0.022299712138935248
  - - 0.03330596438859795
    - 0.0544969966256903
    - -0.9979583108000676
    - -0.05441572852117096
  - - -0.8961708157069298
    - -0.44041600505719586
    - -0.05395935103979155
    - -0.1517325221667203
  - - 0.0
    - 0.0
    - 0.0
    - 1.0

camera_2:
  extrinsic_matrix:
  - - 0.39927490520256714
    - 0.9161722963509223
    - -0.034754474166055815
    - -0.03451909020591836
  - - 0.029708528242828214
    - -0.05081585899988589
    - -0.998266072659864
    - -0.023108206814300713
  - - -0.9163497986168417
    - 0.39755008725087904
    - -0.0475076278264045
    - -0.1716901781403474
  - - 0.0
    - 0.0
    - 0.0
    - 1.0